tox.ini

```diff
@@ -193,6 +194,11 @@ deps =
     celery40,celery41,celery42: pytest>=3,<4
     pytest-mock
     opentracing
+    httpx
+    pytest-asyncio
```

Comment:
Since you are installing these just for starlette tests, you should install it only in `starlette-contrib` env

```suggestion
```

Reply:
I see I'll make this change 👍